src/datatype.rs

```diff
@@ -423,8 +441,9 @@ impl DataType for DataRef<'_> {
 
     fn get_string(&self) -> Option<&str> {
         match self {
-            DataRef::String(v) => Some(&**v),
-            DataRef::SharedString(v) => Some(v),
+            DataRef::String(v) => Some(v),
+            DataRef::RichText(v) => Some(v.text()),
+            DataRef::SharedString(v) => Some(v.text()),
```

Comment:
I'd expect a `SharedString` and ` SharedRichText` ??
What is the issue exactly to force `RichText` all over the place?